tests/autoload.php.dist

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the ni-ju-san CMS.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+// if the bundle is within a symfony project, try to reuse the project's autoload
+
+$files = array(
+    __DIR__.'/../vendor/autoload.php',
+    __DIR__.'/../../vendor/autoload.php',
+    __DIR__.'/../../../../app/autoload.php',
+);
+
+$autoload = false;
+foreach ($files as $file) {
+    if (is_file($file)) {
+        $autoload = include_once $file;
+
+        break;
+    }
+}
+
+if (!$autoload) {
+    die('Unable to find autoload.php file, please use composer to load dependencies:
+
+wget http://getcomposer.org/composer.phar
+php composer.phar install
+
+Visit http://getcomposer.org/ for more information.
+
+');
+}
+g rh
```

Comment:
this makes travis fail

Reply:
strange, but this is not part of the file itself:
https://github.com/core23/ClassificationMediaBundle/blob/master/Tests/tests/autoload.php.dist

Reply:
let's merge and fix afterwards?

Reply:
yes 👍 
will you do it?